src/models/OtrStore.js

```diff
@@ -0,0 +1,67 @@
+var _ = require("lodash");
+var DSA = require('otr').DSA;
+var OTR = require('otr').OTR;
+var Chan = require("./chan");
+
+module.exports = OtrStore;
+
+/** Stores OTR state for a given user
+ *
+ * Handels private key, current sessions and fingerprints.
+ */
+function OtrStore(name, client) {
+	_.merge(this, {
+		name: name,
+		privKey: undefined,
+		sessions: {},
+		client: client,
+	});
```

Comment:
This goes far beyond coding style, and I think I understand why someone would ever want to do that.

```js
function F1(a) {
  this.a = a;
}

function F2(a) {
  require('lodash').merge(this, { a: a });
}

var a = { foo: 'bar' };
var a1 = new F1(a);
var a2 = new F2(a);
console.log(a1.a.foo); // bar2
console.log(a2.a.foo); // bar
```

So doing so (using `_.merge`) deeply clones all objects within the `source` hash :anguished:. Is that really what you want?!
This is really heavy, probably unnecessary in most cases, and most likely harmful in some. Don't get me wrong, I am all for immutable and persistent data structures, but here you get neither of those while potentially messing up your program if you forget objects are not shared. Nice.

By the way, I don't recall seeing any project using lodash/underscore instantiating objects with `merge`, *ever*.
Bottom line is: even if it was only for style (which it is not!), one should not spend half an hour understanding a damn object assignment.